config.example.wikimedia.yaml

```diff
@@ -148,6 +149,85 @@ templates:
                 request:
                   uri: http://graphoid.wikimedia.org/{domain}/v1/png/{title}/{revision}/{graph_id}
 
+      /{module:summary}:
+        x-subspec:
+          info: Summary sys module API
+          paths:
+            /request/{title}:
+              get:
+                x-setup-handler:
+                  - init:
+                      method: 'put'
+                      uri: /{domain}/sys/key_value/summary
+                      body:
+                        valueType: 'json'
+                x-request-handler:
+                  - get_revision:
+                      request:
+                        method: get
+                        uri: /{domain}/sys/page_revisions/page/{title}
+                        headers:
+                          cache-control: '{cache-control}'
```

Comment:
In 99% of the cases this call will come after the content has been refreshed, so we will have the newest rev info in storage already. Hence, I'm not sure it's worth supplying the `cache-control` header here and induce a superfluous call to the MW API.